coq_platform_packages.sh

```diff
@@ -27,39 +27,41 @@ if  [[ "${COQ_PLATFORM_EXTENT}"  =~ ^[fF] ]]
 then
 
 # Some generally useful packages
-PACKAGES="${PACKAGES} coq-unicoq.1.5+8.13"
-PACKAGES="${PACKAGES} coq-ext-lib.0.11.3"
+#PACKAGES="${PACKAGES} coq-unicoq.1.5+8.13"
+#PACKAGES="${PACKAGES} coq-ext-lib.0.11.3"
 PACKAGES="${PACKAGES} coq-equations.1.2.3+8.13"
-PACKAGES="${PACKAGES} coq-bignums.8.13.0"
-PACKAGES="${PACKAGES} coq-aac-tactics.8.13.0"
-PACKAGES="${PACKAGES} coq-mtac2.1.4+8.13"
-PACKAGES="${PACKAGES} coq-simple-io.1.5.0"
-PACKAGES="${PACKAGES} coq-quickchick.1.5.0"
+#PACKAGES="${PACKAGES} coq-bignums.8.13.0"
+#PACKAGES="${PACKAGES} coq-aac-tactics.8.13.0"
+#PACKAGES="${PACKAGES} coq-mtac2.1.4+8.13"
+#PACKAGES="${PACKAGES} coq-simple-io.1.5.0"
+#PACKAGES="${PACKAGES} coq-quickchick.1.5.0"
 
 # Analysis and numerics
-PACKAGES="${PACKAGES} coq-flocq.3.3.1"
-PACKAGES="${PACKAGES} coq-coquelicot.3.1.0"
-PACKAGES="${PACKAGES} coq-gappa.1.4.6 gappa.1.3.5"
-PACKAGES="${PACKAGES} coq-interval.4.1.1"
+#PACKAGES="${PACKAGES} coq-flocq.3.3.1"
+#PACKAGES="${PACKAGES} coq-coquelicot.3.1.0"
+#PACKAGES="${PACKAGES} coq-gappa.1.4.6 gappa.1.3.5"
+#PACKAGES="${PACKAGES} coq-interval.4.1.1"
 
 # Elpi, Coq-elpi and hierarchy builder
-PACKAGES="${PACKAGES} coq-elpi.1.8.1 elpi.1.12.0"
-PACKAGES="${PACKAGES} coq-hierarchy-builder.1.0.0"
+#PACKAGES="${PACKAGES} coq-elpi.1.8.1 elpi.1.12.0"
+#PACKAGES="${PACKAGES} coq-hierarchy-builder.1.0.0"
 
 # The standard set of mathcomp modules
-PACKAGES="${PACKAGES} coq-mathcomp-ssreflect.1.12.0"
-PACKAGES="${PACKAGES} coq-mathcomp-fingroup.1.12.0"
-PACKAGES="${PACKAGES} coq-mathcomp-algebra.1.12.0"
-PACKAGES="${PACKAGES} coq-mathcomp-solvable.1.12.0"
-PACKAGES="${PACKAGES} coq-mathcomp-field.1.12.0"
-PACKAGES="${PACKAGES} coq-mathcomp-character.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-ssreflect.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-fingroup.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-algebra.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-solvable.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-field.1.12.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-character.1.12.0"
 # Plus a few extra mathcomp modules
-PACKAGES="${PACKAGES} coq-mathcomp-bigenough.1.0.0"
-PACKAGES="${PACKAGES} coq-mathcomp-finmap.1.5.1"
-PACKAGES="${PACKAGES} coq-mathcomp-real-closed.1.1.2"
+#PACKAGES="${PACKAGES} coq-mathcomp-bigenough.1.0.0"
+#PACKAGES="${PACKAGES} coq-mathcomp-finmap.1.5.1"
+#PACKAGES="${PACKAGES} coq-mathcomp-real-closed.1.1.2"
+
+PACKAGES="${PACKAGES} coq-hott.dev"
```

Comment:
According to https://github.com/coq/opam-coq-archive/blob/master/extra-dev/packages/coq-hott/coq-hott.dev/opam you need `= "coq.dev"` while here you have `coq.8.13.1`. You can either pick a version of `coq-hott` which works on 8.13, or put your own "overlay coq-hott package" in `./opam/packages/` and tune the deps there.

Reply:
Right!

Reply:
I added a custom made 8.13 and updated the dependency. let's see!